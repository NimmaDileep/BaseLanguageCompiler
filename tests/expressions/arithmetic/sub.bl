var x: Int = 23;
var y: Int = 2;

print(x - y);
print(y - x);

y = -5;

print(x - y);
print(y - x);

var p: Float = 1.5;
var q: Float = 2.7;
print(q - p);
print(x - q);