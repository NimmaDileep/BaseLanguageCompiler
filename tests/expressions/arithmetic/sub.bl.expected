21
-21
28
-28
